priv/repo/migrations/20160226131844_create_stat.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Elide.Repo.Migrations.CreateStat do
+  use Ecto.Migration
+
+  def up do
+    execute "CREATE TYPE stats_tags AS ENUM ('browser','referrer', 'country', 'platform')"
+    create table(:stats) do
+      add :tag, :stats_tags
+      add :value, :string
+      add :count, :integer
+      add :visited_at, :datetime
```

Comment:
Not sure yet! Looking for an option, PG 9.5 has [UPSERT](http://www.postgresql.org/docs/devel/static/sql-insert.html) Ecto doesn't support it yet. I'm thinking that maybe we can a pool up stats which update the counter on at a time for each tag. What do you think?